commands/deploy.go

```diff
@@ -19,6 +19,7 @@ const MaxFileSize = 1073741824 // 1GB
 // DeployCmd handles deploying an app to Section.
 type DeployCmd struct {
 	Debug     bool
+	Directory string `default:"./"`
```

Comment:
Maybe this could work just with a `.`.

Reply:
Yep good call. 

I did some testing, and this works fine. 

Changed in 871ba3c55c15bf5e0cc0db93d6ee633335d1fe8c.